packages/katello/katello/katello-remove

```diff
@@ -3,19 +3,29 @@
 require 'optparse'
 require 'fileutils'
 
+options = {}
 optparse = OptionParser.new do |opts|
-  opts.banner = "This script is used to remove all katello related packages and services. \nThis should only be used if you are sure you are wanting to completly remove Katello from this machine and lose all of your settings and synced content."
+  opts.banner = "This script is used to remove all Katello related packages and services. \nThis should only be used if you are sure you are wanting to completly remove Katello from this machine and lose all of your settings and synced content."
 
+  options[:unattended] = false
+  opts.on('-y', '--assume-yes', 'Remove without confirmation') do
+    options[:unattended] = true
+  end
+
+  opts.on('-h', '--help', 'Displays Help') do
+    puts 'To start removal without confirmation do "katello-remove --unattended", otherwise run "katello-remove" to start the cleanup.'
```

Comment:
OptionParser will automatically generate a help menu, so no need to have the option